install/kubernetes/cilium/crds/crd-servicemonitor.yaml

```diff
@@ -0,0 +1,460 @@
+# https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
```

Comment:
This should be under a conditional branch. This should only be installed if the serviceMonitor is enabled.

Reply:
That is a good point, but possibly not achievable with Helm:

> CRD files cannot be templated. They must be plain YAML documents.

I shall see if there's any way that could make this conditional.

Reply:
:exploding_head: